main.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+require_relative 'decorator'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+require_relative 'person'
+
+person = Person.new(22, 'Maximilianus')
```

Comment:
- [ Required ] I think it would be nice if you call the `Person` class constructor with the same arguments as those present in the `initialize` method in the `Person` class. Currently, when I run the application I get the error below. I would like to suggest that you refactor this line of code as shown below.

## Example code
```ruby
person = Person.new(22, name: 'maximilianus')
```
## Error when running the project
![Screenshot from 2023-10-12 00-16-19](https://github.com/ahmidou09/OOP-school-library/assets/16392046/4136be26-b374-46d5-bfd8-7a2d54374580)